client/my-sites/site-settings/jetpack-module-toggle.jsx

```diff
@@ -31,7 +31,7 @@ class JetpackModuleToggle extends Component {
 	};
 
 	static propTypes = {
-		siteId: PropTypes.number.isRequired,
```

Comment:
Thanks for the clarification. Let's leave it as you suggest and we can always go back to it if necessary.